### ISSUE TITLE
List: Screen readers are unable to announce the single selection case

### ISSUE THREAD

**Issue by sleric1024** [UI5/webcomponents#10351]
### Bug Description

**Accessibility Issue with Screen Readers:**
Screen readers are unable to announce the single selection case. In contrast, for multi-selection, screen readers successfully convey the MultiSelection message to sightless users.

### Expected Behaviour

Both single selection or multiple selection should be recognized by screen readers.

**multi-selection items:**
Contains Multi-Selectable Items.  ✅

**single-selection items:**
Contains Selectable Items  ❌

**Ensure that once the focus is on single or multiple selection section, screenreader should announce the selection mode accordingly.** Such as Contains Single-Selectable Items.

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/List/

### Steps to Reproduce

1.
2.
3.
...

### Priority

None

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Additional Context

This issue occurs from b2411 deliver and is requested from a11y expert.

### Organization

SAP Successfactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by sleric1024** [comment]
Please don't close this because seems the previous one didn't fixed yet. https://github.com/SAP/ui5-webcomponents/issues/7905

**Comment by plamenivanov91** [comment]
Hello @sleric1024 ,

I tested [List Selection Modes Snippet](XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9MaXN0LmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9MaXN0SXRlbVN0YW5kYXJkLmpzXCI7XG5cbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L21hcC5qc1wiOyJ9fQ) and it seems that all of the selection modes are properly announced by JAWS.

Could you please test it as well?

Regards,
Plamen Ivanov

**Comment by sleric1024** [comment]
@plamenivanov91 , thanks for your quick response.
While could you please help to check if this announcement is about the list title?
<img width="1056" alt="image" src="https://github.com/user-attachments/assets/c29ce35c-31dd-4495-8f91-73c868cd2560">

Because in some case, the title can be defined according to real business usage.
Thanks
Eric

**Comment by plamenivanov91** [comment]
Hello @sleric1024 ,

I changed the titles and they are properly announced when focusing some of the list items.

https://github.com/user-attachments/assets/ca96e3ee-af04-4eb5-9d02-c099c1576067

Is this sufficient for you ?

Regards,
Plamen Ivanov

**Comment by sleric1024** [comment]
@plamenivanov91 , thanks for the information. Confirmed with a11y expert. Could you please ask to confirm with UI5 team PM and A11y Expert.
https://github.com/SAP/ui5-webcomponents/blob/e41370f4869fdc7e37f9dc293a2b7519c10bb6ee/packages/main/src/i18n/messagebundle_en.properties#L157

It is possible to change the label to: ARIA_LABEL_LIST_SELECTABLE=Contains Single-Selectable Items.
If so, the message will be clear to 
ARIA_LABEL_LIST_MULTISELECTABLE=Contains Multi-Selectable Items

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Could you please take over?

Regards,
Plamen Ivanov

**Comment by judith9209** [comment]
Would you please let us know the current state? What is the design item?

**Comment by PetyaMarkovaBogdanova** [comment]
Hi @judith9209 ,
the design items, currently on hold, are FIORIDESIGN-14521 and FIORIDESIGN-14522
Best,
Petya Markova.
UI5 Web Components developer

**Comment by judith9209** [comment]
@PetyaMarkovaBogdanova thanks a lot for sharing.
